src/okareo/autogen_logger.py

```diff
@@ -82,11 +88,23 @@ def __init__(
             "new_client",
         ]
 
+    def register_new_model(self, agent_name: str, tags: list[str]) -> None:
+        """Register an agent as a new model in Okareo, and add the model to the group."""
+        new_model = self.okareo.register_model(
+            name=agent_name,
+            tags=tags,
+        )
+        self.okareo.add_model_to_group(self.group, new_model)
+        self.registered_models[agent_name] = new_model
+        print(f"[Okareo] New agent registered under model: {agent_name}")
+
     def start(self) -> str:
         """Start the logger and return the session_id."""
         try:
-            print(f"[Okareo] Started new session with Session ID: {self.session_id}")
```

Comment:
do we want raw print statements here

Reply:
Added a 'verbose' option to suppress most print statements